cmd/patchVolume_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright © 2019 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	_ "fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestPatchVolumeHalevel runs a series of volume patching test
+func TestPatchVolumeHalevel(t *testing.T) {
+	volName := "patchvol"
+	haLevel := 2
+	shared := true
+	var size uint64
+	size = 2
+
+	testPatchVolume(t, volName, haLevel, size, shared)
+}
+
+//Helper to create a volume
+func volCreate(t *testing.T, volName string) {
+	// Create a volume
+	testCreateVolume(t, volName, 1)
+	// Verify that the volume got created
+	assert.True(t, testHasVolume(volName))
+}
+
+//Helper function to cleanup volume created
```

Comment:
Minor: It is better to leave space after //.

Reply:
Fixed.